vllm/config/compilation.py

```diff
@@ -956,43 +958,62 @@ def set_splitting_ops_for_v1(self):
 
         if self.use_inductor_graph_partition:
```

Comment:
This is too complicated, let's simplify:
```
        if self.pass_config.fuse_attn_quant and not self.use_inductor_graph_partition:
            self.set_splitting_ops_for_attn_fusion()
        else:
```

Inductor partition case is the same as the no-attn-fusion case anyway.